### DISCUSSION TITLE
Submissions for Sorobounty Spectacular are closed!

### DISCUSSION THREAD

**Discussion by briwylde08** [stellar/sorobounty-spectacular#2] [Announcements]
We're so excited you're here! Ready to submit your sweet dApp? Here's what ya do:

1. Click the "Discussions" tab at the top of the page
2. Create a "New Discussion"
3. Title your discussion thread
4. Select the “General” category
5. Github Discussions requires you to search for similar discussion threads before creating a new thread.  It's a good idea!  That way, you can see submissions that have already been turned in.  Don't copy from any, but feel free to create something similar.
6. When you're ready, check off “I have done a search for similar discussions.”
7. Add your submission.  Feel free to put it right in the box, or to link to an external resource.
8. Once you have posted, you're done!

Can't wait to see what you've built!

**Comment by richafltr** [comment]
Is it possible to submit more than one discussion thread if someone has a couple of ideas they want to pursue simultaneously? What's the upper limit to these submissions, are they all eligible for the bounty?

**Reply by briwylde08** [reply]
Hi there! You can submit as many as you'd like as long as each submission is for a distinct dapp.

**Comment by earrietadev** [comment]
Does it need to be in English or is ok if I do it in another language? I haven't seen content in Spanish so I thought it might be a good idea

**Reply by briwylde08** [reply]
Spanish content is great! But we'd need an English translation as well.
